Makefile

```diff
@@ -0,0 +1,154 @@
+DESTDIR ?= /
```

Comment:
I guess you will clean the Makefile once you rebase from master.

One think I noticed is that during build I get the warnings from lmdb, which can be silenced with:
```
# Get rid of useless warning in lmdb
CGO_CFLAGS ?= -Wno-implicit-fallthrough -Wno-stringop-overflow
export CGO_CFLAGS
```

But the question is, why this tool requires lmdb in the first place? That we should try to remove as it is a dependency hard to justify.

Reply:
Cleaned up the deps in `mender`: https://github.com/mendersoftware/mender/pull/1406